app/src/ui/notification/new-commits-banner.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { ButtonGroup } from '../lib/button-group'
+import { Button } from '../lib/button'
+import { Ref } from '../lib/ref'
+import { Octicon, OcticonSymbol } from '../octicons'
+import { Branch } from '../../models/branch'
+
+interface INewCommitsBannerProps {
+  /**
+   * The number of commits behind `branch`
+   */
+  readonly commitsBehind: number
+
+  /**
+   * The base branch that is ahead
+   */
+  readonly branch: Branch
```

Comment:
This name is a bit generic, but I'm open to suggestions for a better name

Reply:
I'm going to name this `baseBranch` to keep inline with what dotcom calls the branch that will be merging incoming commits.

<img width="380" alt="screen shot 2018-06-05 at 3 12 32 pm" src="https://user-images.githubusercontent.com/1715082/40978053-ee059eca-68d2-11e8-9e78-2dab010eb52b.png">